app/views/docs/upgrade.phtml

```diff
@@ -69,7 +67,8 @@
 <p>Verify that the <b>STATUS</b> doesn't have any errors and all the <b>appwrite/appwrite</b> containers have version : <?php echo APP_VERSION_STABLE; ?></p>
 
 <h3><a href="/docs/upgrade#run-the-migration" id="migration">Running the Migration</a></h3>
-
+<p>[Note:You do not need to run migration when upgrading to a minor version. For example, going from 0.13.0 to 0.13.2 would not require running migration.] </p>
```

Comment:
There's a notice element we use in our docs, it'll look like this:

```diff
-<p>[Note:You do not need to run migration when upgrading to a minor version. For example, going from 0.13.0 to 0.13.2 would not require running migration.] </p>
+<div class="notice margin-bottom">
+    <h3>Upgrading to a minor version</h3>
+    <p>You do not need to run migration when upgrading to a minor version. For example, going from 0.13.0 to 0.13.2 would not require running migration.</p>
+</div>
```

This will render it in a blue notice like this.
<img width="935" alt="Screen Shot 2022-08-08 at 4 10 15 PM" src="https://user-images.githubusercontent.com/29069505/183505433-33e6015a-45e0-4057-88ed-a88890b5db19.png">